### ISSUE TITLE
TypeError: 'WorkspaceReply' object is not subscriptable

### ISSUE THREAD

**Issue by wpcarro** [lbonn/i3-quickterm#9]
Thanks for making this. I'm getting the following error when I run `./i3-quickterm`.

```
Traceback (most recent call last):
  File "./i3-quickterm", line 252, in <module>
    toggle_quickterm_select(conf)
  File "./i3-quickterm", line 155, in toggle_quickterm_select
    ws, ws_tree = get_current_workspace(conn)
  File "./i3-quickterm", line 141, in get_current_workspace
    ws = [w for w in conn.get_workspaces() if w['focused']][0]
  File "./i3-quickterm", line 141, in <listcomp>
    ws = [w for w in conn.get_workspaces() if w['focused']][0]
TypeError: 'WorkspaceReply' object is not subscriptable
```

python: `Python 3.6.8`
i3: `i3 version 4.16.1 (2019-01-27) © 2009 Michael Stapelberg and contributors`

**Comment by LandingEllipse** [comment]
Hi @wpcarro ,

I'm not the author of i3-quickterm, but this bug looks familiar. Basically the list comprehension in question can yield no matches if no workspaces are considered focused/active (consider that line 141 effectively equates to the intent "get focused workspace"). I've had this happen when e.g. putting Chrome into fullscreen mode (F11) and then trying to fetch the focused workspace. 

I'm assuming this doesn't occur all the time? In either case if you could provide some more context information that would be helpful.

Cheers,
Ariel

**Comment by lbonn** [comment]
Yes, it would definitely be helpful to have a bit more context. I've just tried the scenario @LandingEllipse  described under sway, it didn't crash but had some unexpected effects nonetheless. I'll try on i3 later this week.

Can you also provide the version of i3ipc you're using?

**Comment by wpcarro** [comment]
@lbonn Here is some version information:

`pip3 show i3ipc`:

```
Name: i3ipc
Version: 2.1.1
Summary: An improved Python library to control i3wm and sway
Home-page: https://github.com/altdesktop/i3ipc-python
Author: Tony Crisci
Author-email: tony@dubstepdish.com
License: BSD
Location: /usr/local/google/home/wpcarro/.local/lib/python3.6/site-packages
Requires: python-xlib, enum-compat
Required-by: 
```

`i3 --version`:

```
i3 version 4.16.1 (2019-01-27) © 2009 Michael Stapelberg and contributors
```

**Comment by lbonn** [comment]
Thanks for your report! This confirms that you're likely running into the issues that was fixed in #10: i3-quickterm was not compatible with i3ipc >= 2.0.1.

It should be fixed on master now but feel free to report other issues if you find some.
